examples/pytorch/image-classification/run_image_classification_no_trainer.py

```diff
@@ -175,7 +175,7 @@ def parse_args():
         default="all",
         help=(
             'The integration to report the results and logs to. Supported platforms are `"tensorboard"`,'
-            ' `"wandb"` and `"comet_ml"`. Use `"all"` (default) to report to all integrations.'
+            ' `"wandb"`, `"comet_ml"` and `"neptune"`. Use `"all"` (default) to report to all integrations.'
```

Comment:
Neptune has no integration in Accelerate, so this needs to be reverted, along with all the changes in the other `no_trainer` examples.

Reply:
Ohh. Sure, done. Could you point me what needs to be done in order to support both Trainer and Accelerate? I didn't find anything integrations-related in docs etc.